master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -92,14 +92,6 @@ Calico's manifests assign its components one of two service accounts.
 Depending on your cluster's authorization mode, you'll want to back these
 ServiceAccounts with the neccessary permissions.
```

Comment:
I think it's still useful, and not sure where else it should go right now.

I think we should leave it for now, and potentially re-jig in a future update.